website/docs/core-concepts/components/remote-state.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Component Remote State
+sidebar_position: 8
+sidebar_label: Remote State
+id: remote-state
+---
+
+Component Remote State is used when we need to get the outputs of an [Atmos component](/core-concepts/components),
+provisioned in the same or a different [Atmos stack](/core-concepts/stacks), and use the outputs as inputs to another Atmos component.
+
+:::info
+
+In Atmos, Remote State is implemented by using these modules:
+
+- [terraform-provider-utils](https://github.com/cloudposse/terraform-provider-utils) - The Cloud Posse Terraform Provider for various utilities,
+  including stack configuration management
+
+- [remote-state](https://github.com/cloudposse/terraform-yaml-stack-config/tree/main/modules/remote-state) - Terraform module that loads and processes
+  stack configurations from YAML sources and returns remote state outputs for Terraform components
+
+:::
+
+<br/>
+
+[terraform-provider-utils](https://github.com/cloudposse/terraform-provider-utils) is implemented in [Go](https://go.dev/) and uses Atmos `Go`
+modules to work with [Atmos CLI config](/cli/configuration) and [Atmos stacks](/core-concepts/stacks). The provider processes stack
+configurations to get the final config for an Atmos component in an Atmos stack. The final component config is then used by
+the [remote-state](https://github.com/cloudposse/terraform-yaml-stack-config/tree/main/modules/remote-state) Terraform module to return the remote
+state for the component in the stack.
+
+<br/>
+
+:::info Disambiguation
+
+- **Terraform Component** is a [Terraform Root Module](https://developer.hashicorp.com/terraform/language/modules#the-root-module)
+  that consists of the resources defined in the `.tf` files in a working directory
+  (e.g. [components/terraform/infra/vpc](https://github.com/cloudposse/atmos/tree/master/examples/complete/components/terraform/infra/vpc))
+
+- **Atmos Component** provides configuration (variables and other settings) for a Terraform component and is defined in one or more YAML stack config
+  files (which are called [Atmos stacks](/core-concepts/stacks))
```

Comment:
Not loving this. 
An "Atmos Component" is terraform, helmfile, packer, etc. It should not be conflated with Terraform.